__test__/api/comment-get.test.js

```diff
@@ -85,3 +85,29 @@ describeIfEndpoint(
     });
   }
 );
+
+describeIfEndpoint(
+  "GET",
+  "/v1/comments/:commentId/votes",
+  "GET '/v1/comments/:commentId/votes'",
+  () => {
+    test("Should return a vote to a comment", async () => {
```

Comment:
this is a test for only `status 200 success` and only with no `query parameters`, you also need to test for the following:

- [ ] when unauthorized request `401` by removing `Authorization`
- [ ] when `commentId` not found `404` by using id not in db
- [ ] any other missing tests you notice